tools/deb/envoy_bootstrap_tmpl.json

```diff
@@ -89,13 +89,13 @@
 
     },
     {
-    "name": "xds-grpc",
-    "type": "STRICT_DNS",
-    "connect_timeout": {{ .connect_timeout }},
+    "name": "calico.dikastes",
+    "type": "STATIC",
+    "connect_timeout": { "seconds": 0, "nanos": 5000000},
     "lb_policy": "ROUND_ROBIN",
     "hosts": [
     {
-    "socket_address": {{ .pilot_grpc_address }}
+     "pipe": { "path": "/var/run/dikastes/dikastes.sock" }
```

Comment:
Don't we still need the xds cluster?

Reply:
we don't need the `grpc` interface for xds in 0.6.0 pilot. Which is the version of Pilot that's released only the REST interface.